apps/files/src/components/TransferOwnershipDialogue.vue

```diff
@@ -50,10 +50,13 @@
 					@search="findUserDebounced" />
 			</p>
 			<p>
-				<input type="submit"
-					class="primary"
-					:value="submitButtonText"
+				<NcButton nativeType="submit"
```

Comment:
```suggestion
				<NcButton native-type="submit"
```
Kebab-case for props ;)